test/AppTest/Handler/HomePageHandlerTest.php

```diff
@@ -15,17 +16,25 @@
 
 class HomePageHandlerTest extends TestCase
 {
+    /** @var ContainerInterface */
```

Comment:
It should be `ContainerInterface|ObjectProphecy`.

The same we should have for `$router`.